src/components/create/__snapshots__/create.spec.js.snap

```diff
@@ -1,22 +1,26 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[` 1`] = `
-"
-    border: 1px dashed #99adb6;
-    background-color: transparent;
+Array [
+  "
+  border: 1px dashed #99adb6;
+  background-color: transparent;
     
-    .carbon-icon::before {
-        font-size:16px;
-    }
+  ",
+  ".sc-ifAKCX",
```

Comment:
wouldn't that change every snapshot generation?

Reply:
I don't think so but I will look into see if I can write it without a snapshot test